### PR TITLE
Update docs with references to release-0.8

### PR DESCRIPTION
**Pull request by Amulyam24**
**What this PR does / why we need it**:
Update docs with references to release-0.8

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Update docs with references to release-0.8
```

### CODE REVIEW

**Comment by Amulyam24** [comment]
TO-DO: request netlify for new URL

**Review by mkumatag — APPROVED** [review]
/lgtm

File: docs/book/src/introduction.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
 
 This book documents Cluster API Provider IBM Cloud v0.7. For other versions please see the corresponding documentation:
```
Comment (Prajyot-Parab) [inline docs/book/src/introduction.md:23]:
```suggestion
This book documents Cluster API Provider IBM Cloud v0.8. For other versions please see the corresponding documentation:
```

**Comment by mkumatag** [comment]
> TO-DO: request netlify for new URL

any update on this? @Amulyam24 @Prajyot-Parab who will be approaching to community for this one?

**Comment by Amulyam24** [comment]
> > TO-DO: request netlify for new URL
> 
> any update on this? @Amulyam24 @Prajyot-Parab who will be approaching to community for this one?

@mkumatag, already created a request - https://github.com/kubernetes/org/issues/4983, waiting for action be taken.

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Pull request merged**